client/my-sites/email/email-providers-comparison/index.jsx

```diff
@@ -67,25 +79,33 @@ const identityMap = ( item ) => item;
 
 class EmailProvidersComparison extends React.Component {
 	static propTypes = {
-		domain: PropTypes.object.isRequired,
+		// Props passed to this component
+		selectedDomainName: PropTypes.string.isRequired,
+
+		// Props injected via connect()
+		currencyCode: PropTypes.string,
+		currentRoute: PropTypes.string,
+		domain: PropTypes.object,
+		gSuiteProduct: PropTypes.object,
 		isGSuiteSupported: PropTypes.bool.isRequired,
+		productsList: PropTypes.object.isRequired,
+		selectedSiteId: PropTypes.number,
+		selectedSiteSlug: PropTypes.string,
+		titanMailProduct: PropTypes.object,
 	};
 
 	isMounted = false;
 
 	constructor( props ) {
 		super( props );
 
-		const isDomainEligibleForEmail = this.isDomainEligibleForEmail( props.domain );
-		const hasEmailForwards = this.doesDomainHaveEmailForwards( props.domain );
```

Comment:
I've removed those variables since `props.domain` is not always available when this component is instantiated.